extend/generic-extractor/configuration/aws-signature/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: AWS Signature
+permalink: /extend/generic-extractor/configuration/aws-signature
+---
+
+* TOC
+{:toc}
+
+Generic extractor allows signaturing requests by [**AWS**](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).
+Signature is the process to add authentication information to your requests.
+
+A sample AWS signature configuration looks like this:
+
+{% highlight json %}
+{
+  ...,
+  "aws": {
+    "signature": {
+      "credentials": {
+        "accessKeyId": "testAccessKey",
+        "#secretKey": "testSecretKey",
+        "serviceName": "testService",
+        "regionName": "testRegion"
+      }
+    }
+  }
+}
+{% endhighlight %}
+
+## AWS Signature Credentials
```

Comment:
přidat link na example

Reply:
example je hned nad tím... tam asi není potřeba přidávat odkaz ne?

Reply:
je, abys měl kompletní a config a abys uměl obráceně dohledat, kde je ten example použitej v dokumentaci

Reply:
já asi pořád nerozumím co by si chtěl...to co komentuješ jsou jen popsaný parametry z examplu co je hned nad tím:

<img width="671" alt="image" src="https://user-images.githubusercontent.com/12143866/146826097-84104c41-e9c4-407f-96ad-3fada6e12c0c.png">

a potom v examplu co je v `configuration` má prolink sem

<img width="819" alt="image" src="https://user-images.githubusercontent.com/12143866/146825884-9ee5af4f-e690-445a-86f1-c0709fa26045.png">

viz - https://github.com/keboola/developers-docs/pull/183/files#diff-d2f2793968ecda561bd2ccc068f0c07971c72e4e22cca9efdb59bfc831bb4d1dR17


víc prolinkovat to už asi nejde

Reply:
![image](https://user-images.githubusercontent.com/4319320/146828965-27087dcf-9678-44de-a9bb-e272ea3a2813.png)

Reply:
jo takhle to ty myslíš.... tak to je potom jiná :)

Reply:
https://github.com/keboola/developers-docs/pull/183/commits/39916442f3fa652ff699a581b82331b7071ab374